ffi-cdecl/SDL2_0_decl.c

```diff
@@ -241,3 +241,4 @@ cdecl_const(SDL_PIXELFORMAT_ABGR32)
 
 cdecl_struct(SDL_version)
 cdecl_func(SDL_GetVersion)
+cdecl_func(SDL_SetHint)
```

Comment:
Is this the best position? Not saying it isn't, just want to make sure it's not just added to the end of the file. ;-)

Reply:
That's exactly what I did :p. In both files. Feel free to suggest a better position

Reply:
I think we mostly just take the order from the SDL header files both for easier cross-reference and because they already thought about it, but at a glance I'd say it goes best around here? (But I haven't checked the SDL headers.)

https://github.com/koreader/koreader-base/blob/30adb7538a81b30f78b917f262b58a5a08eded5b/ffi-cdecl/SDL2_0_decl.c#L104-L125